### ISSUE TITLE
T&Cs issues

### ISSUE THREAD

**Issue by matthewdarwin** [canada-ca/PCO-Public-Opinion-Research__Recherche-en-opinion-publique--BCP#3]
Hi,
This project is linked to https://github.com/canada-ca/devex/blob/master/GCDevExchange_-_Terms_and_Conditions.pdf.  This PDF seems to be a template that needs to get filled in?

Also I'm confused that on page 6, section 32 we need to use the laws of BC but on page 11 we're using the laws of Canada.  Which is it?

On page 3 it looks like we're supposed to use email submission, but then on the web page it says we submit online.  Which is it?

**Comment by RachelMuston** [comment]
Hi @matthewdarwin,

Good catches!  Thank you. 

We will be reviewing the Terms and Conditions asap.  We’ll post a message here when we’ve uploaded a new version to the devex repo.

**Comment by mgifford** [comment]
It would be much nicer if you could just post the document in a text version to GitHub so that we can use version control to see what has changed.  Why are we referring to a PDF, other than that this is what paper based governments have always done. 

I know we're not Estonia yet, but ultimately you just need a signature page that points back to a particular version of the text document (as captured in GitHub).

**Comment by RachelMuston** [comment]
But I want to be Estonia!  :-)
Good points.  We are guilty as charged...using the pdf because its how BC did it. 
We also need something that we can easily share with our legal department.  I'm thinking we'll keep a Word doc version off-line and post a .txt version on the repo.  Would that work?

**Comment by mgifford** [comment]
@RachelMuston registration is open https://e-resident.gov.ee/

PDFs are the default for a lot of government agencies.  Using Markdown in GitHub would allow for people to see version control. Any reason it couldn't just be written here in MarkDown & exported to whatever your legal team needs after that?  That's pretty much how things are going with the Open First Whitepaper as I understand it https://github.com/canada-ca/Open_First_Whitepaper

**Comment by RachelMuston** [comment]
Hi @mgifford and @matthewdarwin 

New draft T&Cs are up (in markdown). :)  
https://github.com/canada-ca/devex/blob/master/GCDevEx_TermsandConditions_Dec20_2017.md 

Still to fix is the text related to emailing proposals.  @matthewdarwin, you are correct, that is all done via the web interface now.  Currently drafting text to fix the conditions related to this.

**Comment by steve-h** [comment]
As the initial developer assigned, things worked out relatively well. I will close this issue. A warning to future is that the submitted proposal stopped being visible to me. Glad I had a copy as it is entered online and I saw no "export your proposal" feature.  I would not have known what I had promised to do!

Markdown for the win, or ftw as I am learning,  and even recommending markdown as a better solution for the archives.

Congrats to @mgifford for his recent Estonian credentials, btw.
